init-tools.cmd

```diff
@@ -59,6 +59,12 @@ if exist "%DOTNET_CMD%" goto :afterdotnetrestore
 
 if NOT exist "%DOTNET_PATH%" mkdir "%DOTNET_PATH%"
 
+:: set registry to take dump automatically when test process crashes
```

Comment:
This is going to change people environment. Should we change it to just be done when in CI?

We can detect that via env vars that azure devops sets.

Reply:
Agreed this should only run on CI.

Reply:
good point, will fix this

---
In reply to: [367621559](https://github.com/dotnet/machinelearning/pull/4666#discussion_r367621559) [](ancestors = 367621559)